readthedocs/proxito/README.rst

```diff
@@ -42,82 +42,38 @@ path prefix:
                                     version     |
                                              filename
 
-URL patterns
-------------
+Custom path prefixes
+--------------------
 
-URL patterns define how docs are served, and how the path is parsed by proxito.
-
-For example, the default paths used are:
+By default we serve documentation from the following paths:
 
 - Multi version project: ``/<language>/<version>/<filename>``
 - Single version project: ``/<filename>``
 - Subproject: ``/projects/<subproject-alias>/<subproject-path>``
 
-URL patterns can be used to change these paths per project,
-with the following Project model attributes:
-
-urlpattern:
-   URL pattern to change the path for multi version and single version projects.
-
-urlpattern_subproject:
-   URL pattern to change the path up to the subproject alias (the ``/projects`` prefix).
-   To change the pattern of the subproject itself, use ``urlpattern`` on the subproject.
-
-A URL pattern is a regex with replacement fields,
-replacement fields are expanded to define the proper regex and capture groups (components).
-Valid replacement fields are: ``language``, ``version``, ``filename``, and ``subproject``.
-
-Where to define the custom URL patterns
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-To change the URL pattern of a project,
-define the custom URL pattern in the ``urlpattern`` attribute of the project itself.
-For a translation, change the main language project ``urlpattern``.
-
-And to change the URL pattern of the subproject prefix (``/projects``),
-define the custom URL pattern in the  ``urlpattern_subproject`` attribute of the super project.
-
-Writing a custom URL pattern
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-.. note::
-
-   Custom URL patterns aren't meant to be written by users (yet),
-   a Read the Docs developer needs to do it for the user.
+Custom path prefixes can be used to change the prefix from where the documentation is served from,
+and even change the ``/projects`` prefix for subprojects.
+These prefixes can be changed per project, with the following Project model attributes:
 
-The simplest way to write a custom URL pattern is to start from the default ones,
-and change them as needed.
+custom_prefix:
+   Add a prefix for multi version and single version projects.
 
-- Multi version project: ``/{language}(/({version}(/{filename})?)?)?``
-- Single version project: ``/{filename}``
-- Subproject: ``/projects/{subproject}(/{filename})?``
+custom_subproject_prefix:
+   Change the ``/projects`` prefix for subprojects.
+   To change the prefix of the subproject itself, use ``custom_prefix`` on the subproject.
 
```

Comment:
Is it possible to have a couple examples here to show how to define them?

Reply:
Added one example at https://github.com/readthedocs/readthedocs.org/pull/10156/commits/9d860f8caaf61ca7f8a40d23dd496468868197da